citesphere-model/src/main/java/edu/asu/diging/citesphere/factory/impl/CitationFactory.java

```diff
@@ -272,6 +273,16 @@ private void processGilesUploads(JsonObject jObj, ICitation citation) {
         }
     }
 
+    private void processSameAs(JsonObject jObj, ICitation citation) {
+        citation.setReferences(new HashSet<>());
+        if (jObj.has("sameAs") && !jObj.get("sameAs").isJsonNull()) {
+              List<String> sameAsList = new ArrayList<>();
+              JsonArray sameAsArr = jObj.get("sameAs").getAsJsonArray();
```

Comment:
it's really not that long, add the two last characters as well for readability `sameAsArray`